Modules/ChequeModule/Views/ChequeView.xaml

```diff
@@ -17,17 +18,165 @@
+                <TextBox
+                    Style="{StaticResource CommonTextBoxStyle}"/>
+            </StackPanel>
+        </StackPanel>
+
+        <StackPanel Grid.Row="1" Grid.Column="3" Margin="0,15,0,0" Grid.RowSpan="2" Grid.ColumnSpan="3" >
+            <TextBlock
+                Text="Preview:" 
+                HorizontalAlignment="Center"
+                Style="{StaticResource UserLoginTitleTextStyle}"/>
+        </StackPanel>
+
+        <StackPanel 
+                Grid.Row="1" Grid.Column="3" 
+                HorizontalAlignment="Center"
+                VerticalAlignment="Center" 
+                Height="210" 
+                Width="420"
+                Background="White" 
+                Margin="0,20,5,0" 
+                Grid.RowSpan="4"
+                Grid.ColumnSpan="3">
+            <StackPanel>
+                <Canvas>
+                    <Line
+                        X1="20" Y1="70"
+                        X2="250" Y2="70"
+                        Stroke="Black"
+                        StrokeThickness="0.5" />
+                    <Line
+                        X1="20" Y1="95"
+                        X2="250" Y2="95"
+                        Stroke="Black"
+                        StrokeThickness="0.5" />
+                    <Line
+                        X1="20" Y1="120"
+                        X2="250" Y2="120"
+                        Stroke="Black"
+                        StrokeThickness="0.5" />
+                    <Line
+                        X1="200" Y1="170"
+                        X2="400" Y2="170"
+                        Stroke="Black"
+                        StrokeThickness="0.5" />
+                    <Rectangle
+                        Width="140"
+                        Height="35"
+                        Fill="White"
+                        Stroke="Black"
+                        StrokeThickness="2"
+                        Canvas.Left="260"
+                        Canvas.Top="76" HorizontalAlignment="Left" VerticalAlignment="Top"/>
+                </Canvas>
+            </StackPanel>
+
+            <Grid>
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="auto"/>
+                    <ColumnDefinition Width="17*"/>
+                    <ColumnDefinition Width="11*"/>
+                </Grid.ColumnDefinitions>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="48"/>
+                    <RowDefinition Height="*"/>
+                    <RowDefinition Height="*"/>
+                    <RowDefinition Height="*"/>
+                </Grid.RowDefinitions>
+                <TextBlock Grid.Row="1" Grid.Column="0" 
+                    Text="PAYEE" 
+                    Margin="22,10,173,1" FontSize="9" IsEnabled="False" Grid.ColumnSpan="1" FontStyle="Italic"/>
+
+                <TextBlock Grid.Row="2" Grid.Column="0" 
+                    Text="RINGGIT MALAYSIA" 
```

Comment:
I have researched some of the cheque samples, if we use the `RINGGIT MALAYSIA` will limited to only one country. I think this can be an selective options by combobox. Just a heads up, this will be another work item.